modules/hub/pages/hub-api-installation.adoc

```diff
@@ -0,0 +1,21 @@
+[[hub-api-installation]]
+= Hub XMRPC API Installation
+
+
+Install the package hub-xmlrpc-api, available from SUSE Manager 4.1 and Uyuni repos, either on the Hub itself or on a host that has access to all Servers' XMLRPC APIs.
+
+Configuration of `hub-xmlrpc-api` is specified in a JSON file like the following:
```

Comment:
Are there defaults for the values below? Ie. will it work if I set nothing at all?